site/app/controllers/admin/ReportController.php

```diff
@@ -631,6 +631,8 @@ public function generateCustomization() {
                 'display_benchmarks' => $customization->getDisplayBenchmarks(),
                 'benchmark_percents' => (array) $customization->getBenchmarkPercent(),
                 'benchmarks_with_input_fields' => ['lowest_a-', 'lowest_b-', 'lowest_c-', 'lowest_d'],
+                'final_benchmarks_with_input_fields' => ["A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D"],
```

Comment:
shouldn't this be something like `final_cutoffs_benchmark_input`?

Reply:
refactored names in [c51856b](https://github.com/Submitty/Submitty/pull/10518/commits/c51856bc71f31aa7c3439c4fc83bc345c3f0e652) and [3d90329](https://github.com/Submitty/Submitty/pull/10518/commits/3d903293118ff15a2f64ed912a7eff757ce40ecd)